docs/DeveloperGuide.md

```diff
@@ -72,10 +72,16 @@ The **API** of this component is specified in [`Ui.java`](https://github.com/AY2
 
 ![Structure of the UI Component](images/UiClassDiagram.png)
```

Comment:
Perhaps the Ui class diagram could be re-rendered without the attribute and method boxes?